docs/workflow-controller-configmap.yaml

```
@@ -228,6 +228,8 @@ data:
     # Use a self-signed cert for TLS
     # >= 3.6: default true
     secure: true
+    # Which temporality to use for opentelemetry, defaults to Cumulative
```

```suggestion
    # >= 3.6. Which temporality to use for OpenTelemetry. Default is "Cumulative"
```

- add version number
- use "Default is 'x'" like the other comments here

---

An OTel link describing temporality would probably be helpful here too